pkg/apis/ndm/register.go

```diff
@@ -1,5 +1,5 @@
 package ndm
 
 const (
-	GroupName = "openebs.io"
+	GroupName = "ndm"
```

Comment:
what is this change for?

Reply:
we will discuss on it. directory structure should be like `pkg/apis/openebs.io/` as the groupName is `openebs.io`